odk1-src/central-projects.rst

```diff
@@ -130,12 +130,20 @@ To manage App Users for a Project, you can navigate to the Project overview page
 Archiving a Project
 ~~~~~~~~~~~~~~~~~~~
 
-When you Archive a Project, the following things will happen:
+When you Archive a Project, the following things become frozen:
+
+ - Any Forms that are still in :guilabel:`Open` or :guilabel:`Closing` states will remain in those states.
+ - Any users who are Project Managers will retain their access.
+ - All Project data, including Forms, Form Attachments, and all Submission data will remain available for viewing, download, and access over OData.
```

Comment:
The way this is phrased, it sounds like viewing/download is only available via OData. Maybe
> All Project data, including Forms, Form Attachments, and all Submission data will remain available for viewing and download. Data will also remain accessible via OData.

Reply:
“viewing, download, and OData access” ?

Reply:
Your tweak sounds good to me.